public_html/components/com_jed/models/extension.php

```diff
@@ -18,4 +20,279 @@
  */
 class JedModelExtension extends BaseDatabaseModel
 {
+	/**
+	 * Method to get a specific extension.
+	 *
+	 * @param   integer  $pk  The id of the primary key.
+	 *
+	 * @return  object|boolean|JException  Extension data object on success, boolean false or JException instance on error
+	 *
+	 * @since   4.0.0
+	 */
+	public function getItem($pk = null): object
```

Comment:
Is there a reason why we are not calling the parent function here? After the parent returns the item we add any extra info to the object.

Reply:
Frontend `BaseDatabaseModel` has no parent